tests/fixtures_jlap.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (C) 2012 Anaconda, Inc
```

Comment:
As I said above this should live in `conda.testing.gateways.fixtures` and `conda.testing.gateways.jlap`

Reply:
I'm scarred by my time in Java where everything is in its own individual tiny file, but we can move these around.

Reply:
That's fair, but I think it would still be good since we made an effort to introduce `conda.testing` only recently to make solver development easier.

Reply:
Interesting you should say that, every time I come across that module I have trouble understanding why it's not in tests/

Reply:
`conda.testing` exists so we can reuse testing features in other packages such as conda-libmamba-solver. This was only recently introduced to make it easier to have consistent test environments.

Reply:
Maybe it needs a big comment at the top of the file or something.